packages/firebase/useAuth/demo.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import firebase from 'firebase/app'
+
+import { computed } from 'vue'
+import { stringify } from '@vueuse/docs-utils'
+import { useAuth } from '.'
+import { reactive } from 'vue-demi'
+
+if (!firebase.apps.length) {
+  firebase.initializeApp({
+    apiKey: 'AIzaSyBcgy5ezJoYC1Ed7uwstgy9x5pR5nrv6Q8',
+    authDomain: 'vueuse-auth.firebaseapp.com',
```

Comment:
Yeah it was just one I created for the demo, can easily be changed if you'd like

Reply:
This could potentially bring some privacy issues, are you good with moving the code to the `readme.md` to have a "Full demo" code snippet instead of runnable demo? Or do you have any better solution for it?

Reply:
Agreed, I'll delete the firebase app and we can move it to just the demo. Although I don't know if it is that much of a privacy concern as these are things that would normally be shipped in a client side app I believe and they are restricted by domains. Either way just for simplicity we can move the demo to the docs